server_src/settings/serverSettings.js

```diff
@@ -108,5 +108,5 @@ class ServerSettings {
 
 module.exports = {
   ServerSettings,
-  filamentManagerEnabled: filamentManager
+  filamentManager
```

Comment:
Me reverting my half-assed attempt at renaming a setting. I only foresee issues if this is not done all the way thru the UI, database etc. Thats why I reverted it.

Reply:
Yeah that's a lot of places xD